include/crocoddyl/core/integrator/rk4.hpp

```diff
@@ -94,24 +99,32 @@ struct IntegratedActionDataRK4Tpl : public ActionDataAbstractTpl<_Scalar> {
     }
 
     dx = VectorXs::Zero(ndx);
+    u_diff = std::vector<VectorXs>(4, VectorXs::Zero(nu_diff));
```

Comment:
In Euler I could remove it because I could use the `w` variable of the controlParametrizationData. However, in RK4 I cannot do the same unless I create 4 instances of controlParametrizationData. At the moment I have only one instance of it.